src/components/Activities/Activity3/Game/FeaturesSlidebar.tsx

```diff
@@ -0,0 +1,156 @@
+      case 3:
+        adjustWeights3(100 - weight2 - weight1 + adjustment);
+    }
+  };
+
+  const adjustWeights1 = (val: number) => {
+    if (val < 0 || val > 100) return;
+    const newWeight2 = Math.max(weight2 + weight1 - val, 0);
+    setWeight1(val);
+    setWeight2(newWeight2);
+  };
+
+  const adjustWeights2 = (val: number) => {//val is percent that weight2 takes, not net position
+    if (val > 100) return;
+    const upperBound = val + weight1;
+    if (upperBound > 100) {
+      const newWeight1 = weight1 - (upperBound - 100);
+      setWeight1(newWeight1);
+    } else if (val < 0) {
+      const newWeight1 = Math.max(weight1 + val, 0);
+      setWeight2(0);
+      setWeight1(newWeight1);
+      return;
+    }
+    setWeight2(val);
+  };
+
+  const adjustWeights3 = (val: number) => {//val is percent that weight3 takes, not net position
+    if (val < 0 || val > 100) return;
+    let newWeight2 = 100 - val - weight1;
+    if (newWeight2 < 0) {
+      const newWeight1 = weight1 + newWeight2;
+      newWeight2 = 0;
+      setWeight1(newWeight1);
+    }
+    setWeight2(newWeight2);
+  };
+
+  return (
+    <>
+      <p>
+        Now that we&apos;ve decided which priorities to use, are any of these priorities more important than the others?
+      </p>
+      <p>
+        Adjust the slider so that the most important variables take up the most space.
+      </p>
+      <div className='slider-text-input-container'>
+        <div className='input-option'>
+          <div className='input-text-display-container'>
+            <div className='variable-image' id={variableSelection[0].toLowerCase().replace(' ', '-')} />
+            <div className='feature-text'>{variableSelection[0]}</div>
+          </div>
+          <input className='slider-input'
+            type='text'
+            value={weight1 + '%'}
+            onWheel={scrollValue(1)}
+            onChange={handleInputWeight(1)}
+          />
+        </div>
+        <div className='input-option'>
+          <div className='input-text-display-container'>
+            <div className='variable-image' id={variableSelection[1].toLowerCase().replace(' ', '-')} />
+            <div className='feature-text'>{variableSelection[1]}</div>
+          </div>
+          <input className='slider-input'
+            type='text'
+            value={(weight2) + '%'}
+            onChange={handleInputWeight(2)}
+            onWheel={scrollValue(2)}
+          />
+        </div>
+        <div className='input-option'>
+          <div className='input-text-display-container'>
+            <div className='variable-image' id={variableSelection[2].toLowerCase().replace(' ', '-')} />
+            <div className='feature-text'>{variableSelection[2]}</div>
+          </div>
+          <input className='slider-input'
+            type='text'
+            value={(100 - weight2 -weight1) + '%'}
+            onWheel={scrollValue(3)}
+            onChange={handleInputWeight(3)}
+          />
+        </div>
```

Comment:
beware when you are copying and pasting--something is probably not right
```suggestion
 {
          variableSelection.map((variable, index) => {
            return <div className='input-option'>
            <div className='input-text-display-container'>
              <div className='variable-image' id={variable.toLowerCase().replace(' ', '-')} />
              <div className='feature-text'>{variable}</div>
            </div>
            <input className='slider-input'
              type='text'
              value={weights[index] + '%'}
              onWheel={scrollValue(index)}
              onChange={handleInputWeight(index)}
            />
          </div>;
          })
        }
```